hw/ip/hmac/dv/README.md

```diff
@@ -69,12 +69,14 @@ virtual sequence is extended from `cip_base_vseq` and serves as a starting point
 All test sequences are extended from `hmac_base_vseq`. It provides commonly used handles,
 variables, functions and tasks that the test sequences can simple use / call.
 Some of the most commonly used tasks / functions are as follows:
-* `hmac_init`     : initialize hmac settings including configurations and interrupt
+* `hmac_init`       : initialize hmac settings including configurations and interrupt
   enables
-* `csr_rd_digest` : read digest values from the CSR registers
-* `wr_key`        : write key values into the CSR registers
-* `wr_msg`        : write messages into the hmac_msg_fifo
-* `compare_digest`: compare the read digest result with the expected values
+* `csr_rd_digest`   : read digest values from the CSR registers
+* `wr_key`          : write key values into the CSR registers
+* `wr_msg`          : write messages into the hmac_msg_fifo
+* `burst_wr_msg`    : burst write messages into the hmac_msg_fifo
+* `compare_digest`  : compare the read digest result with the expected values
+* `save_and_restore`: execute Save and Restore as described in the [SW Guide](https://opentitan.org/book/hw/ip/hmac/doc/programmers_guide.html#saving-and-restoring-the-context)
 
 ##### Standard test vectors
 Besides constrained random test sequences, hmac test sequences also includes [standard
```

Comment:
Besides constrained random test sequences, hmac test sequences also include [standard
SHA-2 256 and HMAC test vectors](../../../dv/sv/test_vectors/README.md) from
[NIST](https://csrc.nist.gov/Projects/Cryptographic-Algorithm-Validation-Program/Secure-Hashing#shavs)
and [IETF](https://tools.ietf.org/html/rfc4868) to test SHA-2 256/384/512 and HMAC using SHA-2 256 with 256-bit keys. The standard test vectors provide messages, keys (for the keyed HMAC tests only), and expected
results. The expected results are used to verify the DUT outputs.

Reply:
Why only `HMAC using SHA-2 256 with 256-bit keys` and not also 384-512 bit keys?

Reply:
This just reports on the current state. As of now, we have tests using test vectors for HMAC based on SHA-2 with 256-bit keys. In upcoming PRs, all digest sizes X key lengths will be supported.

Reply:
Maybe I should add a TODO and an issue link, otherwise I am sure we'll forget to add it. What do you think?

Reply:
Yes, agree, we can use this issue https://github.com/lowRISC/opentitan/issues/22932 in here and track in the issue that we need to update the DV documentation. I will be using this same issue  for TODO in this https://github.com/lowRISC/opentitan/pull/23541as well where necessary.

I'd rather not create more spin-off issues than we need.